usr/share/rear/restore/NFS_SERVER/default/400_restore_with_nfs_server.sh

```diff
@@ -0,0 +1,25 @@
+# 400_restore_with_nfs_server.sh
+
+FILE=$TARGET_FS_ROOT/rear_finished.txt
```

Comment:
Oh, and if you want to use a 100% conflict-free filename then I'd suggest `/mnt/rear-restore-done` because `/mnt` is excluded by default

Reply:
@schlomo wouldn't it be better to use `/var/lib/rear/$FILE` ? This use of `/mnt` is not very consistent with the intended/usual purpose of this directory. If it becomes a configuration item, excluding it will be easy: it should be added to the default `BACKUP_PROG_EXCLUDE` (we already have `"$VAR_DIR/output/*"` there).
Also, where do you see that `/mnt` is excluded by default? I see `/mnt` in `AUTOEXCLUDE_PATH` which does not seem to be the same thing (it excludes filesystems with mountpoints under `/mnt` but not files under `/mnt` directly).

Reply:
@pcahyna the signal file is deleted before waiting for the restore and immediately after that, so we should only worry about a potential conflict with an existing file in the system. As ReaR doesn't perform the backup in this case, we don't need to handle excluding anything (that would be the job of the backup software that is used).

Personally I find `/mnt` safe enough and users can change it if they like. Also the current setting of `/rear-restore-done` is fine for me, as it seems also very unlikely to conflict.

Reply:
> As ReaR doesn't perform the backup in this case, we don't need to handle excluding anything

I would not make this assumption, IMO it is perfectly valid to restore this way a backup that has been stored to the NFS client using `rear mkbackup[only]` with BACKUP=NETFS or BACKUP=RSYNC (cf. your "Implement asymmetric solutions, using one tool for backup and another tool for restore, in this case via NFS".)

>  (that would be the job of the backup software that is used)

Indeed, but the backup software may be ReaR itself (see above), so it makes sense to provide a sensible configuration at least for those who use this case.

> Personally I find /mnt safe enough and users can change it if they like. 

I think it is safe, but not necessarily the best choice - if ReaR has a space for its files under `/var/lib/rear`, it seems preferable to use this (or use `/` as before, since we are restoring the content of `/`, it makes sense to put the file there).

Reply:
As we are only discussing the defaults I'm fine with going forward and setting `/var/lib/rear/rear_finished.txt` as the default value. @codefritzel can you please adjust accordingly?